deployment/README.md

```diff
@@ -2,7 +2,7 @@
 
 Deployment config for caikit embeddings.
 
-## Deploy the service
+## Deploy the Service
```

Comment:
FYI - IBM writing style usually suggests "sentence style" here so it would be a lowercase s in Service.  But you'll see "title style" like this used a lot as well so just being consistent here looks fine.